### ISSUE TITLE
model: Use pre-trained models

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#152]
Investigate Tensorflow hub and pytorch hub. Maybe create a model which only knows the accuracy and predict methods.

**Comment by pdxjohnny** [comment]
We now have support for both
